python_modules/dagster-graphql/dagster_graphql/schema/runs.py

```diff
@@ -73,17 +74,22 @@ class Meta:
 
 class GrapheneLaunchRunResult(graphene.Union):
     class Meta:
-        from dagster_graphql.schema.backfill import pipeline_execution_error_types
```

Comment:
why was this removed?

Reply:
and why was it there in the first place 🤔

Reply:
Not sure but I moved the import to top of file: https://github.com/dagster-io/dagster/pull/25880/files#diff-d6694e6d32ed9271ea5c36951e11ae6f342d707257fb8056db67e50df6d69d14R14